crates/router/src/connector/powertranz/transformers.rs

```diff
@@ -101,9 +102,24 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for PowertranzPaymentsRequest
     fn try_from(item: &types::PaymentsAuthorizeRouterData) -> Result<Self, Self::Error> {
         let source = match item.request.payment_method_data.clone() {
             api::PaymentMethodData::Card(card) => Ok(Source::from(&card)),
-            _ => Err(errors::ConnectorError::NotImplemented(
-                "Payment method".to_string(),
-            )),
+            api::PaymentMethodData::CardRedirect(_)
```

Comment:
```suggestion
            api::PaymentMethodData::Wallet(_)
            | api::PaymentMethodData::CardRedirect(_)
```
Here you haven't include wallet enum. This is the reason for clippy check to fail.